src/commonstyles.css

```diff
@@ -0,0 +1,12 @@
+html,
+body {
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+    font-family: Helvetica, sans-serif;
+}
+
+.app-container{
```

Comment:
Create a separate file for component style. Every component should have it's own css file and should be imported in component's file. Only common style rules should be here.